tests/src/types.test.ts

```
@@ -0,0 +1,170 @@
+/**
+ * Tests for ZigEVM type conversions and operations
+ * 
+ * These tests verify that the TypeScript bindings correctly handle
+ * the primitive types used in ZigEVM, such as U256, Address, etc.
+ */
+import { describe, it, expect, beforeAll, afterAll } from 'vitest';
+import path from 'path';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)